airflow_entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
+
+if [ "$#" -lt 1 ]; then
+    echo "Please add the command you would like to run with Airflow."
+    exit
+fi
+
+/wait
+
+if [ "$1" != "initdb" ]; then
+    sleep 5
```

Comment:
arbitrary sleep is always a smell, what is this waiting for? `/wait` or sequencing the parts of `command:` with `&&` should address all of that?

Reply:
Add a comment explaining why, so that if/when we go to production we can implement a proper waiting condition on `postgres` being initialized with tables.

Reply:
comments added.